draft-thomson-saag-sig-context.md

```diff
@@ -137,10 +138,49 @@ zero-valued octet at the end can be registered on a First-Come, First-Served
 basis [RFC5226].  Context strings that do not meet these requirements require
 Expert Review [RFC5226].
 
+The initial contents of this registry are included in {{ok}}.
+
 
 # Security Considerations
 
-Derp, derp, derp.
+In general, it is best to restrict the use of any cryptographc material to a
+single use.
```

Comment:
I think "to being used for a single purpose" might be more clear than this, which could be read to imply that crypto material is single-use in the sense of a one-time pad.